docs/user-guide/gaffer-basics/what-is-aggregation.md

```diff
@@ -0,0 +1,70 @@
+# What is Aggregation?
+
+**Aggregation** - noun
+
+*"The formation of a number of things into a cluster"*
+
+---
+
+In a software context Aggregation can have a variety of interpretations, in
+Gaffer this specifically refers to the aggregation function. This takes
+a number of forms but the common factor between them is that they use the
+underlying koryphe library to provide the
+[`ElementAggregator`](https://gchq.github.io/Gaffer/uk/gov/gchq/gaffer/data/element/function/ElementAggregator.html).
```

Comment:
This feels very much like an implementation detail which isn't essential reading for a user. I don't think there's really another place to put it (potentially the aggregation overview page); maybe it could go in a collapsible section?